src/lib/EventPublisher.js

```diff
@@ -57,8 +88,10 @@ class EventPublisher extends EventEmitter {
       ttl,
     };
 
-    this._emitWithPrefix(eventData.name, event);
-    this.emit(ALL_EVENTS, event);
+    this._publish(event);
```

Comment:
So this definitely works but I think it would make sense to only call this if the process is master or if `_useCluster` is false. Then you could remove the `eventOwnerWorker.id === worker.id`

We can talk about this in chat if it makes sense or not.